utils/hare-bootstrap

```diff
@@ -411,7 +411,12 @@ done
 echo 'Consul ready on all nodes'
 
 say 'Updating Consul configuraton from the KV store...'
-update-consul-conf --server &
+runs_server=''
```

Comment:
Do we need this for remote execution of update-consul-conf as well at line #427?